po/common/common.pot

```diff
@@ -6172,3 +6172,24 @@ msgid "Nutri-Score, Eco-Score and food processing level (NOVA)"
 msgstr "Nutri-Score, Eco-Score and food processing level (NOVA)"
 
 
+msgctxt "actions_add_ingredients"
+msgid "Could you add the ingredients list?"
+msgstr "Could you add the ingredients list?"
+
+msgctxt "actions_to_compute_nutriscore"
+msgid "Could you add the information needed to compute the Nutri-Score?"
+msgstr "Could you add the information needed to compute the Nutri-Score?"
+
+msgctxt "action_add_ingredients_text"
+msgid "Add the ingredients"
+msgstr "Add the ingredients"
+
+msgctxt "action_add_categories"
+msgid "Add a category"
+msgstr "Add a category"
+
+msgctxt "action_add_nutrition_facts"
+msgid "Add nutrition facts"
+msgstr "Add nutrition facts"
+
```

Comment:
missing : "action_edit_nutrition"

Reply:
I'll remove the action and just keep "add_nutrition_facts"